uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -98,14 +97,16 @@ define([
           amount: 0,
           maxAmount: 0,
           disabled: true,
-          loading: false
+          loading: false,
+          skip_value_setting: false
         },
         to: {
           time: '',
           amount: 0,
           maxAmount: 0,
           disabled: true,
-          loading: false
+          loading: false,
+          skip_value_setting: false
```

Comment:
Shouldn't it be `skipValueSetting` ?

Reply:
I thought vars should be skipValueSetting and props skip_value_setting?

Reply:
Fixed anyway.